<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>up</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="6" column="0">
      <widget class="QPushButton" name="add_images_file_btn">
       <property name="text">
        <string>Add Image</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="delete_images_file_btn">
       <property name="text">
        <string>Delete Image</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="3" colspan="2">
      <widget class="QPushButton" name="close_btn">
       <property name="text">
        <string>Save and Close</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="5" colspan="5">
      <widget class="QListWidget" name="images_path_list_widget">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
